api/controllers/product.js

```diff
@@ -69,12 +37,22 @@ async function queryAllProduct() {
     console.error(err)
     throw new Error(err)
   });
-  return tempProduct
+  return res.response(tempProduct)
 }
 
 module.exports.getAllProduct = async (req, res) => {
-  const product = await queryAllProduct()
-  return res.response(product)
+  const tempProduct = await product.findAll({
+    include: [{
+      model: category
+    }, {
+      model: seasonal
+    }
+    ]
```

Comment:
please mind indentations, more readable
```suggestion
    include: [
      { model: category },
      { model: seasonal }
    ]
```